R/fortify-spatial.R

```diff
@@ -9,32 +9,31 @@
 #' @param ... not used by this method
 #' @keywords internal
 #' @name fortify.sp
-#' @examples
-#' if (require("maptools")) {
-#'  sids <- system.file("shapes/sids.shp", package="maptools")
-#'  nc1 <- readShapePoly(sids,
-#'    proj4string = CRS("+proj=longlat +datum=NAD27"))
-#'  nc1_df <- fortify(nc1)
-#' }
 NULL
 
 #' @rdname fortify.sp
 #' @export
 #' @method fortify SpatialPolygonsDataFrame
 fortify.SpatialPolygonsDataFrame <- function(model, data, region = NULL, ...) {
+  deprecate_warn0("3.4.3",
```

Comment:
Thanks for handling this! If the version is going to be 3.4.4, maybe we should reflect here?

Reply:
Yes, for sure. This branch is just for testing; if we merge the maptools retirement into the RC branch we'll put in the proper version number.

Reply:
I see. Thanks for the details!